packages/hub/src/scope.ts

```diff
@@ -174,6 +191,32 @@ export class Scope implements ScopeInterface {
     return this;
   }
 
+  /**
+   * @inheritDoc
+   */
+  public setSpanContext(spanContext?: SpanContext): this {
```

Comment:
Slight naming inconsistency. This one is `setSpanContext`, which accepts `SpanContext`, but the name of this thing is `_span`, plus all other methods also use just `Span`.

We don't use `Context` anywhere else, like `setUserContext` `getUserContext` etc. We should keep with one naming convention.